src/main/java/carsharing/configuration/DBManager.java

```diff
@@ -0,0 +1,60 @@
+package carsharing.configuration;
+
+import lombok.NoArgsConstructor;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+@NoArgsConstructor
+public class DBManager {
+
+    private static final String CREATE_TABLE_COMPANY = """
+            CREATE TABLE IF NOT EXISTS company
+            (id INTEGER PRIMARY KEY AUTO_INCREMENT,
+            name VARCHAR(100) UNIQUE NOT NULL);""";
+
+    private static final String CREATE_TABLE_CAR = """
+            CREATE TABLE IF NOT EXISTS car
+            (id INTEGER PRIMARY KEY AUTO_INCREMENT, name VARCHAR(100) UNIQUE NOT NULL,
+            company_id INTEGER NOT NULL,
+            CONSTRAINT fk_company FOREIGN KEY(company_id)
+            REFERENCES company(id));""";
+
+    private static final String CREATE_TABLE_CUSTOMER = """
+            CREATE TABLE IF NOT EXISTS customer
+            (id INTEGER PRIMARY KEY AUTO_INCREMENT, name VARCHAR(100) UNIQUE NOT NULL,
+            rented_car_id INTEGER DEFAULT NULL,
+            CONSTRAINT fk_car FOREIGN KEY(rented_car_id)
+            REFERENCES car(id));
+            """;
+
+    protected static String DB_URL;
```

Comment:
Why do we need to change state of static field?

Reply:
DB_URL = "jdbc:h2:./src/main/java/carsharing/db/" + name; 
We can change db name